testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers/src/main/java/org/keycloak/testsuite/rest/TestingResourceProvider.java

```diff
@@ -824,6 +824,59 @@ public TestJavascriptResource getJavascriptResource() {
         return new TestJavascriptResource();
     }
 
+    @POST
+    @Path("/enable-feature/{feature}")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response enableFeature(@PathParam("feature") String feature) {
+
+        Profile.Feature featureProfile = Profile.Feature.valueOf(feature);
+
+        if (featureProfile == null || Profile.isFeatureEnabled(featureProfile))
```

Comment:
IMO in case that Profile.isFeatureEnabled(featureProfile) is true (which means that feature is already enabled), it will be better to just return OK response. That would be in-par to the "disableFeature()" method below, which just returns OK response in case that feature is already disabled. So I suggest to use same behaviour in "enableFeature" as well.

Reply:
You're right. I thought this approach is correct, because the test should verify that the feature is disabled by default, so I did it. But it's true, that might be better way, how to verify that. Thanks!